Ravel-Creators/Assets/RavelCreator/Scripts/Editor/RavelEditor.cs

```diff
@@ -245,7 +256,7 @@ public static class RavelEditor
         float m = mask.width / mask.height;
         float t = (float)tex.width / tex.height;
 
-        if (Mathf.Abs(m - t) <= MathBuddy.FloatingPoints.LABDA) {
+        if (Mathf.Abs(m - t) <= FloatingPoints.LABDA) {
             return coords;
         }
         if (m > t) {
```

Comment:
Delete duplicate DrawTextureScaleHeightGUI()

Reply:
This is not a duplicate. One method is `width` the other is `height`. 

It is not used (right now), but the height version scales the width of the image based on the heigth and the width version scales the height, based on the width.

I feel like this method may be needed at some later point in creating the editor codes.